flink-runtime-web/web-dashboard/app/partials/jobs/job.plan.jade

```diff
@@ -28,10 +28,10 @@ split
             a(ui-sref=".subtasks({nodeid: nodeid})") Subtasks
 
           li(ui-sref-active='active')
-            a(ui-sref=".taskmanagers({nodeid: nodeid})") TaskManagers
+            a(ui-sref=".taskmanagers({nodeid: nodeid})") Subtasks by TaskManager
```

Comment:
How about merging the two tabs, and adding a checkbox to "aggregate by TaskManager"?